README.md

```diff
@@ -18,9 +18,9 @@ Sinker is built by [Paradigm](https://www.paradigm.xyz/), and is licensed under
 
 In Postgres, you might have a normalized schema like this:
 
-- A Student and a Teacher refer to a Person
+- A Student and a Teacher refers to a Person
 - A Course is taught by a Teacher
-- Students have and belong to many Courses through the Enrollment join table
+- Students have taken and belong to many Courses through the Enrollment join table
```

Comment:
I'd prefer to keep this language as HABTM is a standard way to indicate a many-to-many relationship.